scripts/stage-docs.sh

```diff
@@ -45,6 +49,16 @@ if [[ -z ${RELEASE_VERSION} ]]; then
   exit 1
 fi
 
+if [[ -z ${RELEASE_DOCS_BRANCH} ]]; then
+  echo "No release branch specified, aborting"
+  exit 1
+fi
+
+if [[ -z ${RUN_ID_LABEL} ]]; then
```

Comment:
Is there a reason this is assumed to be an env var rather than being passed in through the cli?

Making it an arg would be consistent with the other flags and with `stage-release.sh`

Reply:
That's dead code.  I'd originally thought to label the docs PR with the run id too.  As discussed above, I didn't need it.

Reply:
Removed.